ml_tools/eolearn/ml_tools/tdigest.py

```diff
@@ -121,8 +121,8 @@ def _is_output_ftype(feature_type: FeatureType, mode: ModeTypes, pixelwise: bool
 
 
 def _looper(
-    in_feature: list[FeatureSpec], out_feature: list[FeatureSpec], eopatch: EOPatch
-) -> Generator[tuple[FeatureSpec, FeatureSpec, np.ndarray], None, None]:
+    in_feature: list[tuple[FeatureType, str]], out_feature: list[tuple[FeatureType, str]], eopatch: EOPatch
+) -> Generator[tuple[tuple[FeatureType, str], tuple[FeatureType, str], np.ndarray], None, None]:
```

Comment:
since it repeats four times in a very limited space, perhaps it would make sense to define a local type alias

Reply:
we can replace it if we decide to keep a `Feature` alias, otherwise it can be as it is, it's an extra anyway